demisto_sdk/commands/test_content/tests/BuildContext_test.py

```diff
@@ -295,3 +298,23 @@ def test_playbook_with_version_mismatch_is_skipped(mocker, tmp_path):
                                              content_conf_json=content_conf_json,
                                              filtered_tests_content=filtered_tests)
     assert 'playbook_with_version_mismatch' in build_context.tests_data_keeper.skipped_tests
+
+
+def test_unmockable_playbook_configuration(mocker, tmp_path):
```

Comment:
do we have test for playbook that is not set as `is_mockable` (the default scenario) and see it does not come up in the `unmockable_test_ids` ?

Reply:
I'll add one.